config_cedar.go

```diff
@@ -15,7 +15,10 @@ type CedarConfig struct {
 	User        string `bson:"user" json:"user" yaml:"user"`
 	APIKey      string `bson:"api_key" json:"api_key" yaml:"api_key"`
 	// Insecure disables TLS, this should only be used for testing.
-	Insecure bool `bson:"insecure" json:"insecure" yaml:"insecure"`
+	Insecure     bool   `bson:"insecure" json:"insecure" yaml:"insecure"`
```

Comment:
Which of these values are going to go away after the cutover? If anything is staying, could you add comments so we understand long-term?

Reply:
Added some comments calling out the ones that are going away